### PR TITLE
resolves issue #21 by documenting flow testing

### PR DESCRIPTION
**Pull request by mmroden**
Adds in tables etc for the flow testing

### CODE REVIEW

File: content/en/build-guide/fan-box.adoc [hunk 1]
```diff
@@ -26,7 +26,7 @@ image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-cn4M
 
 == Assembly Diagram == 
 
-{{< embed-pdf hidePaginator=true url="/D-TET-001103-1 Fan Box Drawing.pdf" >}}
+{{< embed-pdf hidePaginator=true url="./D-TET-001103-1 Fan Box Drawing.pdf" >}}
```
Comment (capsulecorplab) [inline content/en/build-guide/fan-box.adoc:28]: drawing renders fine, but isn't clickable

File: content/en/build-guide/m12.adoc [hunk 2]
```diff
@@ -23,7 +23,11 @@ image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC
 
 == Assembly Diagram == 
 
-{{< embed-pdf hidePaginator=true url="/D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+{{< embed-pdf hidePaginator=true url="./D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
```
Comment (capsulecorplab) [inline content/en/build-guide/m12.adoc]: drawing renders fine, but isn't clickable

File: content/en/build-guide/m12.adoc [hunk 3]
```diff
@@ -23,7 +23,11 @@ image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC
...
+
+== Circuit Diagram == 
+
+{{< embed-pdf hidePaginator=true url="./D-TET-001100-0 Papra PCBA Drawing.pdf" >}}
```
Comment (capsulecorplab) [inline content/en/build-guide/m12.adoc]: Drawing not rendering for me Screenshot from 2021-05-17 08-59-16

File: content/en/build-guide/m12.adoc [hunk 4]
```diff
@@ -23,7 +23,11 @@ image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC
 
 == Assembly Diagram == 
 
-{{< embed-pdf hidePaginator=true url="/D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+{{< embed-pdf hidePaginator=true url="./D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+
+== Circuit Diagram == 
+
```
Comment (capsulecorplab) [inline content/en/build-guide/m12.adoc]: There's a previous/next button being generated above the PCBA drawing that doesn't seem to do anything (see next comment)

File: content/en/build-guide/mask.adoc [hunk 5]
```diff
@@ -21,7 +21,7 @@ image::https://photos.smugmug.com/Tetra-Testing/Tetra-PAPRa-Build-Party-31-Jan-2
 
 == Assembly Diagram
 
-{{< embed-pdf hidePaginator=true url="/D-TET-001104-0 Mask Assembly Drawing.pdf" >}}
+{{< embed-pdf hidePaginator=true url="./D-TET-001104-0 Mask Assembly Drawing.pdf" >}}
```
Comment (capsulecorplab) [inline content/en/build-guide/mask.adoc:24]: drawing renders fine, but isn't clickable

File: content/en/testing-guide/_index.adoc [hunk 6]
```diff
@@ -1,65 +1,26 @@
...
-The other hose, the blue hose, is left exposed to the air to provide a sample of the ambient air to the Portacount.
-
-Here are the videos of the various tests performed:
-
-video::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-Kq8Lp8G/0/37bf76f3/1920/IMG_5191-1920.mp4[]
+We came up with a series of tests to ensure that the PAPRa would be performant.  Performant means:
 
-Testing the fit of the mask itself:
+1.  When worn, the wearer received N99 levels of protection. link:testing_march_2021[This page] describes how we did that testing in March of 2021 on both individual fan units and on a tester wearing the entire PAPRa.
+2.  The unit must provide a minimum amount of flow.  Hooded PAPRs require a flow of 6 CFM.  Since the PAPRa is designed to provide airflow for a half facepiece, the flow requirement is 4 CFM, or 113.27 L/min.  As of link:flow_testing_may_2021[May 2021], PAPRas provide 67-92 L/min of airflow, depending on hose configuration, so more work is necessary to bring the units up to this flow specification[1].
```
Comment (capsulecorplab) [inline content/en/testing-guide/_index.adoc]: Specs should be reviewed by a subject matter expert

File: content/en/testing-guide/_index.adoc [hunk 7]
```diff
@@ -1,65 +1,26 @@
...
 
-video::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-GgKRbwK/0/a835042a/1920/IMG_5188-1920.mp4[]
+References:
 
-As you can see, while these tests are promising, they are not yet completely rigorous.  We are currently working on a full PAPR testing system, and will likely also reach out to third party testing labs for further verification of individual units.
+1. link:https://multimedia.3m.com/mws/media/378601O/interpretation-of-inhalation-airflow-measurements.pdf[Interpretation of Inhalation Airflow Measurements for Respirator Design and Testing. L.L. Janssen, N.J. Anderson, P.E. Cassidy, R.A. Weber, and T.J. Nelson. Journal of the International Society for Respiratory Protection, Vol. 22, pp. 122-141] referencing Burgess WA, Reist PC (1969) Supply rates for powered air purifying respirators.   Am Ind Hyg Assoc J 30:1-6.
```
Comment (capsulecorplab) [inline content/en/testing-guide/_index.adoc]: I would convert this to a [footnote](https://docs.asciidoctor.org/asciidoc/latest/macros/footnote/)

File: content/en/testing-guide/flow_testing_may_2021.adoc [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+1.  OW (Orange top, white funnel, no weather shield)
+2.  GB (Grey top, black funnel)
+
+The battery units were:
+1. OG (Orange casing, grey potentiometer)
+2. GG (Grey casing, grey potentiometer)
+3. GB (Grey casing, black potentiometer)
+
+The potentiometers were set to L (low, just turned on) and H (high, turned to the maximum flow).
```
Comment (capsulecorplab) [inline content/en/testing-guide/flow_testing_may_2021.adoc:58]: Is there a P/N call out to this potentiometer somewhere? If so, we should probably reference it

**Review by capsulecorplab — CHANGES_REQUESTED** [review]
added review notes on pdf renderings, sections that need review by an SME, & footnote usage

File: content/en/testing-guide/flow_testing_may_2021.adoc (hunk 8 above)
Comment (capsulecorplab) [reply content/en/testing-guide/flow_testing_may_2021.adoc:58]: consider rewording _potentiometer_ to _flow control knob_ or something descriptive to the user

File: content/en/build-guide/fan-box.adoc (hunk 1 above)
Comment (mmroden) [reply content/en/build-guide/fan-box.adoc:28]: Honestly not sure why that is; the clickability seems to be related to the embedding, and maybe that doesn't work anymore due to that other library updating?  From #105 Bump lodash from 4.17.19 to 4.17.21?  I doubt it.  Will check it out.

File: content/en/build-guide/m12.adoc (hunk 2 above)
Comment (mmroden) [reply content/en/build-guide/m12.adoc]: as above

Comment (mmroden) [reply content/en/build-guide/m12.adoc]: no pdfs are currently clickable, not just the ones I changed.

File: content/en/build-guide/m12.adoc (hunk 3 above)
Comment (mmroden) [reply content/en/build-guide/m12.adoc]: Should not have a second pdf render on the same page, the embedding can't handle it.

File: content/en/build-guide/m12.adoc (hunk 4 above)
Comment (mmroden) [reply content/en/build-guide/m12.adoc]: Should be fixed

File: content/en/build-guide/mask.adoc (hunk 5 above)
Comment (mmroden) [reply content/en/build-guide/mask.adoc:24]: larger problem

File: content/en/testing-guide/_index.adoc (hunk 7 above)
Comment (mmroden) [reply content/en/testing-guide/_index.adoc]: done

File: content/en/build-guide/fan-box.adoc (hunk 1 above)
Comment (mmroden) [reply content/en/build-guide/fan-box.adoc:28]: BTW, our current site does not allow clicking, so I'm thinking this is a separate issue.

File: content/en/build-guide/m12.adoc (hunk 3 above)
Comment (mmroden) [reply content/en/build-guide/m12.adoc]: should be fixed now

File: content/en/testing-guide/_index.adoc (hunk 6 above)
Comment (mmroden) [reply content/en/testing-guide/_index.adoc]: Not sure what we need here-- ask Daniel, maybe?  He's told me that these are reasonable goals.

Comment (capsulecorplab) [reply content/en/testing-guide/_index.adoc]: Will mark it as resolved then

File: content/en/build-guide/fan-box.adoc (hunk 1 above)
Comment (capsulecorplab) [reply content/en/build-guide/fan-box.adoc:28]: opened an issue https://github.com/tetrabiodistributed/papra/issues/23

File: content/en/build-guide/m12.adoc (hunk 3 above)
Comment (capsulecorplab) [reply content/en/build-guide/m12.adoc]: fix looks good now

File: content/en/testing-guide/_index.adoc (hunk 7 above)
Comment (capsulecorplab) [reply content/en/testing-guide/_index.adoc]: looks good

**Review by capsulecorplab — APPROVED** [review]
LGTM

**Pull request merged**